pkg/ddc/base/runtime.go

```diff
@@ -73,7 +74,7 @@ type RuntimeInfoInterface interface {
 
 	GetRuntimeType() string
 
-	IsExclusive() bool
```

Comment:
How about adding a function ? It can expose the initialization state without forcing every caller to perform a nil check on the pointer returned by IsExclusive()
```
func (info *RuntimeInfo) HasDatasetInfo() bool {
    return info.exclusive != nil
}
```